meta-balena-common/recipes-core/balena-rollback/files/rollback-health

```diff
@@ -18,6 +18,7 @@ set -o errexit
 
 . /usr/sbin/balena-config-vars
 . /usr/bin/rollback-parse-bootloader
+. /usr/libexec/os-helpers-tpm2
```

Comment:
@jakogut not all devices types will include `os-helpers-tpm2`, to make it future proof check for the file existence before sourcing it.